src/lesson 13/script.js

```diff
@@ -0,0 +1,260 @@
+const input = document.querySelector('#start_seconds');
+const btnTimeout = document.querySelector('#start_timeout');
+
+const userSeconds = () => {
+    alert('The End!');
+}
+
+btnTimeout.addEventListener('click', function() {
+    const timer = setTimeout(userSeconds, input.value * 1000);
+
+    setTimeout(() => { clearInterval(timer)}, (input + 1) * 1000);
```

Comment:
Nothing happens after we enter the value 

https://user-images.githubusercontent.com/84010501/199959525-ce2a4b71-2948-4a0d-a180-748203c372a2.mov